tests/client/amo/testApi.js

```diff
@@ -204,20 +216,21 @@ describe('amo.api', () => {
         .expects('callApi')
         .returns(Promise.resolve({ results: [] }));
 
-      return getLatestUserReview({ user: 123, addon: 321 })
+      return getLatestUserReview({ user: 123, addon: 321, version: 456 })
         .then((review) => {
           assert.strictEqual(review, null);
         });
     });
 
-    it('requires a user and addon', () => {
+    it('requires a user, addon, and version', () => {
```

Comment:
Technically we only changed the error message and don't actually discretely test for each case–it would be nice to have three tests here, each one making sure that two out of three params still triggers the error.